src/Northstar.php

```diff
@@ -51,11 +51,13 @@ public function getAllUsers($inputs = [])
      * Send a GET request to return a user with that id.
      *
      * @param string $id
+     * @param array $fields - optional fields to request
      * @return NorthstarUser
      */
-    public function getUser($id)
+    public function getUser($id, $fields = null)
     {
-        $response = $this->get('v2/users/'.$id);
+        $include = $fields ? implode(',', $fields) : null;
+        $response = $this->get('v2/users/'.$id, ['include' => $include]);
```

Comment:
So empty or `null` param values are automatically dropped? Is that a Guzzle thing?

Reply:
Yep!